packages/graphql/mixin.core.js

```diff
@@ -109,6 +114,17 @@ class GraphQLMixin extends Mixin {
       );
     }
   }
+
+  diagnose() {
+    if (!existsSync(this.config.fragmentsFile)) {
+      return [
```

Comment:
We might want to use [`submitDiagnose`](https://github.com/untool/untool/blob/master/packages/info/lib/diagnoses/generic.js#L11) for use cases such as this.

Reply:
Why? This should be displayed as a warning.
Or do you mean we should use that hook to be able to print a warning and a hint?
As in:

```javascript
  diagnose({ submitDiagnosis }) {
    if (!existsSync(this.config.fragmentsFile)) {
      const logger = this.getLogger();
      submitDiagnosis(
        logger.warning.bind(logger),
        `Could not find a graphql introspection query result at "${
          this.config.fragmentsFile
        }".`
      );
      submitDiagnosis(
        logger.hint.bind(logger),
        'You might need to execute "hops graphql introspect"'
      );
    }
  }
```